config.mk

```diff
@@ -0,0 +1,3 @@
+registry=amrc-factoryplus.shef.ac.uk:5000
```

Comment:
Can we replace this with a placeholder? Alternatively, would it not be better practice to add `config.mk` to `.gitignore` and include ` config.mk.example`?

Reply:
Ack, that was supposed to be gitignored altogether. It isn't required for a vanilla build, it's an optional override. I'll remove it.